docs/words/examples/oscardemo.py

```diff
@@ -46,7 +46,8 @@ def gotBuddyList(self, l):
         self.setIdleTime(0)
         self.clientReady()
         self.createChat('%s Chat'%SN).addCallback(self.createdRoom)
-    def createdRoom(self, (exchange, fullName, instance)):
+    def createdRoom(self, exchange_fullname_instance):
```

Comment:
this is a callback, so exchange_fullname_instance should be called just `result`